src/mame/drivers/alphatpx.cpp

```diff
@@ -1,6 +1,6 @@
 // license: GPL-2.0+
 // copyright-holders: Dirk Best
-// test / modify oct-2017 begin: rfka01 + helwie44
```

Comment:
Can you please not make non-standard heading comments like this?  We actually scrape these things for various purposes.

If you think you've made a significant copyrightable contribution to this file, add your name to the `copyright-holders:` heading.  (Facts are not copyrightable, i.e. things like memory maps, PCB layouts, graphics decode macros, etc.)

We recognise a `thanks-to:` heading for significant contributions that are not eligible for copyright protection.

For anything else, just add notes in the big multiline comment below.